html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+      <div>
+        <form action="index.html">
+          <p>Email: <br>
+            <input type="text" name="email" size="15" maxlength="30" />
+          </p>
+        </form>
+      </div>
+      <div>
+        <form action="index.html">
```

Comment:
There should be only one `<form>` element used. All of the inputs for this exercise should be within a single form.